presto-main/src/main/java/com/facebook/presto/SessionRepresentation.java

```diff
@@ -67,6 +67,7 @@
     private final Map<String, SelectedRole> roles;
     private final Map<String, String> preparedStatements;
     private final Map<SqlFunctionId, SqlInvokedFunction> sessionFunctions;
+    private final Map<String, String> customHeaders;
```

Comment:
Can you help me understand why we need changes to this class.
Are we burning in the passed in headers into the session. Is that required ?

Reply:
I did it as in AbstractTestingPrestoClient 
ClientSession is created from Session class fields which is affecting the SessionRepresentation. 
I can leave it ImmutableMap.empty() if you think is better?

Reply:
yup.. lets not change Session for this. cc @tdcmeehan

Reply:
Removed it